app/frontend_messages.py

```diff
@@ -0,0 +1,15 @@
+from flask.ext.babel import gettext as _
+
+
+MESSAGES = {
+            'multiple-surveys': {
+                'TITLE': _("Information"),
+                'MSG': _("Unfortunately you can only complete one survey at a time."),
```

Comment:
Additionally, shouldn't we be adding all existing messages to this structure rather than creating something new?